app/controllers/citizens/own_homes_controller.rb

```diff
@@ -17,8 +17,8 @@ def update
     private
 
     def form_params
-      merge_with_model(legal_aid_application) do
-        return {} unless params[:legal_aid_application]
+      merge_with_model(legal_aid_application, journey: :citizens) do
+        next {} unless params[:legal_aid_application]
 
```

Comment:
I don't understand why `return` has been replaced with `next` here (or in the other controllers.

Reply:
When 'return' is used the translation is looking for the 'shared_ownership.blank' translation for the page and not the journey specific one, e.g shared_ownership.citizens.blank, this pattern is also used on vehicles_controller and success_likely_controller to achieve the same result. from what I read online , next exits the block and returns its argument as block return value, which in this case includes the 'journey'. I am not sure why return does not also include the 'journey' however.